test/active_record_test.rb

```diff
@@ -1,6 +1,7 @@
 require_relative './test_helper'
 
 class I18nBackendActiveRecordTest < I18n::TestCase
+
   def setup
```

Comment:
let's not add an empty line here
```suggestion
  def setup
```